Report.md

```diff
@@ -164,27 +160,21 @@ These diagrams represents the dynamic behaviour of objects and the structure of
 
 ## **Possible Effects of Project Failure**
-If the information system is ineffective, lacklustre or fundamentally useless, there are some considerable adverse effects to first take into account. Ranging from the loss of customers, as opposed to sharp gain, to increased operating costs for no benefit, the adverse effects possible should be weighed into consideration alongside the competitive advantages to most accurately see the possible result of the information system.
+If the information system is ineffective, there are some considerable adverse effects to first take into account. Ranging from the loss of customers to increased operating costs for no benefit, the adverse effects possible should be weighed into consideration alongside the competitive advantages to most accurately see the possible result of the information system.
 
-By implementing what the travel company believes is a solution to their biggest issues, and communicating to users as such, there is an inherent risk if we fail to deliver. By producing a suboptimal solution, customers may feel rightly disappointed, as their expectations were not met. Particularly if these issues have been ongoing for some time, customers who thought the end of these issues were near may be especially upset and cease their interactions with the travel company. Not only does this result in fewer profits for the company, but customers will most likely still need their services met and will hence move to one of the company's direct competitors - a huge competitive disadvantage.
+By communicating to users that the travel company is implementing a solution to their biggest issues, there is an inherent risk if we fail to deliver. If the solution fails to meet expectation, customers may feel rightly disappointed. If these issues have been ongoing, those who expected an end to their problems may be especially upset and cease interaction with the travel company. There is then a high risk these customers will move to one of the company's direct competitors - a huge competitive disadvantage.
 
-Not only this, but if the system does not work as intended, then the increase in operations (more tasks are performed as a result of the information system) may in fact work against the CMC and slow its efficiency, for little to no reward. This would produce a compounded negative effect, wherein not only is the CMC slower but it is expected to be faster. As such, the lengthened wait times and reduced call flow handling would ultimately lead to greater inbound call costs.
+Additionally, there is a small risk that the increase in tasks performed by CMC may slow its efficiency. This would provide a compounded effect, where the CMC is not only slower but expected to be faster too. This lower efficiency will lead to higher inbound call costs and reduced call flow handling.
```

Comment:
Additionally, there is a small risk that the increase in tasks performed by CMC may decrease its effectiveness. This would have a run-on effect whereby despite the CMC being newly implemented it will be not only run slower but have less efficiency leading to higher inbound call costs and reduced call flow handling. (Just reworded it a bit)

Reply:
BTW You can also suggest changes with that ± button. Then they can be committed/changed from the comment

Reply:
The CMC isn't newly implemented, right? I'll still take on the other feedback

Reply:
I'm not entirely sold either. My point in the second sentence was that they expected to improve its effectiveness but actually brought it down so the problem is twice as bad. Which, you know, can be cut I'm not attached to it or anything ahaha
Just double-checking that's the direction you think it should go

Reply:
Yea definitely not newly implemented probably should've said improved. But what you have definitely works its all good